app/views/layouts/_phase_banner.html.haml

```diff
@@ -1,7 +1,10 @@
-.govuk-phase-banner
+.govuk-phase-banner{ class: ENV['ENV'] }
```

Comment:
This new class isn't needed, we can either put it once on the body or the header.  
In FALA, it was on the body, so the CSS would be the same.

Reply:
👍 thanks - have moved class to body as suggested